docs/admin/README.md

```diff
@@ -8,7 +8,7 @@
  - [`flowforge.yml` configuration](../install/configuration.md) - base platform configuration, done before you run.
  - [First Run Setup](../install/first-run.md) - create your admin user
  - [FlowForge Concepts](../user/concepts.md)
- - [Usage Telemetry](./telemetry.md)
+ - [Usage Telemetry](/docs/admin/telemetry.md)
```

Comment:
Any reason to go to a fully qualified path here?

Reply:
There shouldn't be - but if you checkout the "Usage Telemetry" link on the live site (https://flowforge.com/docs/admin/) - it doesn't work. Locally, this was the only way I could get it to run. It was very strange